rainbow-cake-core/src/main/java/co/zsmb/rainbowcake/config/RainbowCakeConfigurator.kt

```diff
@@ -17,8 +17,26 @@ public interface RainbowCakeConfigurator {
      *
      * Default value: false.
      */
+    @Deprecated(
+        message = "This configuration will be removed. Use isLoggable instead.",
+        replaceWith = ReplaceWith(
+            expression = "isLoggable",
+            imports = ["co.zsmb.rainbowcake.config.RainbowCakeConfigurator"]
+        )
+    )
     public var isDebug: Boolean
 
+    /**
+     * Whether the application is currently loggable or not.
+     * Should almost always take the value of BuildConfig.DEBUG.
+     *
+     * Effects:
+     * - Logging is disabled in non-debug mode, regardless of the [logger] set.
+     *
+     * Default value: false.
+     */
+    public var isLoggable: Boolean
```

Comment:
As `isDebug` was only used for logs, it's more readable to use directly a new `isLoggable` variable.